ScriptureRenderingPipeline/Webhook.cs

```diff
@@ -101,6 +101,27 @@ public static async Task<IActionResult> RunAsync(
 
 			return new OkResult();
 		}
+		// Now I need a function that will listen on /merge and then post to the merge service bus message

```

Comment:
I recommend rewording this comment as documentation -- maybe just remove the "Now I need a function that will" part.  :)